packages/app-store/zoomvideo/api/callback.ts

```diff
@@ -28,7 +28,9 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
     try {
       const responseBody = await result.json();
       errorMessage = responseBody.error;
-    } catch (e) {}
+    } catch (e) {
+      errorMessage = await result.clone().text();
```

Comment:
Why do we need to clone here?

Reply:
Response is really strict about consuming it. So, text() or json() consume it and it can't be re-consumed.

So, we have to be sure that we aren't re-consuming.